test/performance/channel-imc/continuous/100-channel-imc-setup.yaml

```diff
@@ -0,0 +1,103 @@
+# Copyright 2019 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+
+apiVersion: v1
+kind: Service
+metadata:
+  name: channel-imc-consumer
+  namespace: default
+spec:
+  selector:
+    role: channel-imc-consumer
+  ports:
+    - protocol: TCP
+      port: 80
+      targetPort: cloudevents
+      name: http
+
+---
+
+apiVersion: v1
+kind: Service
+metadata:
+  name: channel-imc-aggregator
+  namespace: default
+spec:
+  selector:
+    role: channel-imc-aggregator
+  ports:
+    - name: grpc
+      port: 10000
+      targetPort: grpc
+      protocol: TCP
+
+---
+
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: perf-eventing
+  namespace: default
+
+---
+
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: perf-eventing
+rules:
+  - apiGroups: [""]
+    resources: ["nodes", "pods"]
+    verbs: ["list"]
+
+---
+
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: perf-eventing
+subjects:
+  - kind: ServiceAccount
+    name: perf-eventing
+    namespace: default
+roleRef:
+  kind: ClusterRole
+  name: perf-eventing
+  apiGroup: rbac.authorization.k8s.io
+
+---
+
+apiVersion: messaging.knative.dev/v1alpha1
+kind: InMemoryChannel
+metadata:
+  name: channel-imc
+  namespace: default
+
+---
+
+
+apiVersion: messaging.knative.dev/v1alpha1
+kind: Subscription
+metadata:
+  name: channel-imc-sub
+  namespace: default
+spec:
+  channel:
+    apiVersion: messaging.knative.dev/v1alpha1
+    kind: InMemoryChannel
+    name: channel-imc
+  subscriber:
+    uri: "http://channel-imc-consumer.default.svc.cluster.local"
```

Comment:
Can we set it to a kubernetes service, like https://github.com/knative/eventing/blob/b2e44b5cb15e6ac101ccbea2558e08acca2ec308/test/performance/broker-imc/continuous/100-broker-imc-setup.yaml#L35 ?

Reply:
done